fendermint/app/src/app.rs

```diff
@@ -693,14 +688,17 @@ where
 
     /// Apply a transaction to the application's state.
     async fn deliver_tx(&self, request: request::DeliverTx) -> AbciResult<response::DeliverTx> {
+        tracing::info!("deliver_tx");
```

Comment:
Do we want to keep this one as info? Should we add some context information that we may have available in this context like maybe the block hash or the height?

Reply:
No no, left it by accident. There's not much context we can say unless we log inside where we have the height from the exec_state.